.github/workflows/rust.yml

```diff
@@ -26,14 +26,14 @@ jobs:
         run: cargo fmt -- --check
 
       - name: Build (release) (default features)
-        run: cargo build --target i686-pc-windows-msvc --locked --release
+        run: cargo build -Z build-std --target i686-win7-windows-msvc --locked --release
```

Comment:
For those unaware of why we are building std and thus using a nightly toolchain,
https://github.com/rust-lang/rust/pull/118150